src/NUnitFramework/framework/Constraints/MultipleOfConstraint.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+
+using System;
+
+namespace NUnit.Framework.Constraints
+{
+    /// <summary>
+    /// EvenConstraint tests that the actual value is an even integer
```

Comment:
```suggestion
    /// MultipleOfConstraint tests that the actual value is a multiple of a specified integer.
```

Reply:
Thanks, I initially had separate Even and Odd constraints until I changed them to be instantiations of MultipleOfConstraint instead.